cetmix_tower_server/tests/test_key.py

```diff
@@ -27,33 +27,24 @@ def test_key_creation(self):
             "Trailing and leading whitespaces must be removed from name",
         )
 
-    def test_key_access_rights(self):
-        """Test private key security features"""
+    def test_global_key_access_rights(self):
+        """Test global private key security features"""
 
-        # Default message returned instead of key value
-        SECRET_VALUE_PLACEHOLDER = self.Key.SECRET_VALUE_PLACEHOLDER
-
-        # Store key value
+        # Store global key value
         self.write_and_invalidate(self.key_1, **{"secret_value": "pepe"})
 
         # Get key value as Bob
         key_bob = self.key_1.with_user(self.user_bob)
 
         with self.assertRaises(AccessError):
-            key_value = key_bob.secret_value
+            key_value = key_bob.read(["secret_value"])
 
         # Add user to group
         self.add_to_group(self.user_bob, "cetmix_tower_server.group_user")
 
-        # Get value
-        key_value = key_bob.secret_value
-
-        # Ensure placeholder is used instead of the key value
-        self.assertEqual(
-            key_value,
-            SECRET_VALUE_PLACEHOLDER,
```

Comment:
We need to ensure that SECRET_VALUE_PLACEHOLDER is returned properly., so this test cannot be removed.
If you can modify it or add another one if this one is not working in the new flow.